AdminUi/src/AdminUi/Program.cs

```diff
@@ -155,6 +156,9 @@ static void Configure(WebApplication app)
     {
         ResponseWriter = HealthCheckWriter.WriteResponse
     });
+
+    var eventBus = app.Services.GetRequiredService<IEventBus>();
+    eventBus.StartConsuming();
```

Comment:
Actually, we don't register for push notifications in the Admin UI. So maybe we should remove this call, in order to avoid an open, unused connection to the event bus.

Reply:
That's true... 🤔

Though I think that we will rather create a completely new service that sits next to Consumer API and Admin UI which is responsible for processing all integration events. Listening to events in the Admin UI would probably make things very complicated.

Reply:
But: props for bringing this up :)